typescript/types.ts

```diff
@@ -192,8 +192,12 @@ export type ExecuteResult = {
   /**
    * The result of executing the prompt.
    */
-  data: JSONValue;
-
+  data:
+    | JSONValue
```

Comment:
Long-term, do we want to support arbitrary JSONValue here or just let them dump things into metadata so we can always have a kind/value?

Reply:
We can decide that. My 2c are it's still good to support arbitrary JSONValue here, but we can revisit that. It will require a breaking change to the schema too, so we can consider this for v2 schema